### ISSUE TITLE
More Intensive Posting Guidelines

### ISSUE THREAD

**Issue by cyh76507707** [Steemhunt/web#139]
[Problems]
- Some of the hunts are not suitable for our standard of "cool new product," but it is difficult to not to approve.
- There are too many hunts everyday, so it reduces users to focus on one hunt. More priority of our community is that the single hunt can have more observation and discussion with many hunters. Too many hunting posts disturbs this priority.

[Solutions - More Strict Product LInk]
- We won't accept all e-commerce site such as Amazon, Ali Express, Ebay, etc.
- Only four product links are valid: Official product website, App store link (iOS and Android), Kickstarter and Indiegogo.
- Hunter must put up the official website as the main priority. If the product does not have the main website and only have app store or Kickstarter link, then it is fine with it.
- Moderator must guide them to put up the official website as the first priority.
- The official website must be written in proper English, and made by modern website standards like HTML5. Older standard (like HTML 4 or Adobe flash) will be checked and can be hidden by moderators. 

[Possible Benefits]
- We expect this new policy can reduce more than 40% hunting posts.
- The quality of products posted on SH are likely to be increased.
- We don't need to consider to limit daily hunting per hunter per day because this policy will naturally limit the opportunity.
- This will reduce the chance of duplication problems.

**Comment by Folken8** [comment]
So they are people out there that are still using Adobe Flash? 
wtf. 

I absolutely agree with this improvement.

**Comment by sydneyitguy** [comment]
chronocrypto said on Discord
```
2. Types of Products to Post
Steemhunt generally covers unique IT or hardware products, including:

Web services
Mobile apps
Games
API, IT solutions, bots, open sources or other types of software
Tech gadgets
Unique items
Can we press this a bit more.
I am seeing strollers, lamps, washingmachines... etc ridiculous.
The last one  "Unique items" Honestly should be taken off.
some one posted salt shakers they called them unique...
I also agree amazon should be taken completely off.
```

**Comment by Folken8** [comment]
@sydneyitguy yeah I'm planning to improve this part, will make a pull request tonight probably

**Comment by Folken8** [comment]
@Rishi556 the definition of "cool product" is different from a person to another.

**Comment by cyh76507707** [comment]
Let's delete Unique Items.
Also, Let's add the product screenshot
1) If it is online-based product - It must be captured in proper manner (like full screen shot, high resolution, clear product image...)
2) If it is gadget - It must be from the official images, NOT the images hunter took by his or her device.
